compiler/rustc_mir/src/const_eval/machine.rs

```diff
@@ -309,6 +309,35 @@ impl<'mir, 'tcx> interpret::Machine<'mir, 'tcx> for CompileTimeInterpreter<'mir,
                 );
                 ecx.write_scalar(Scalar::Ptr(ptr), dest)?;
             }
+            sym::panic_ctfe_hook => {
+                // Option<&str>
+                assert!(args.len() == 1);
+
+                debug!("panic_ctfe_hook: invoked with {:?}", args[0]);
+
+                let (_, idx) = ecx.read_discriminant(&args[0])?;
+                let some = match idx.index() {
+                    0 => {
+                        // None
+                        // For now, this is unreachable code - you cannot construct a non-literal
+                        // `fmt::Arguments` without `impl const Display`, which we don't currently
+                        // provide.
```

Comment:
I can't parse this sentence. What does the last "which" refer to?

Reply:
To the `impl const Display`

Reply:
So maybe ", and we currently do not provide such an `impl`"?

But I also don't see what this has to do with `impl const Display`...

Reply:
Actually I think my main confusion is around the term "non-literal `fmt::Arguments`". You also use it in the `panic_ctfe_hook` doc comment, but I do not know what you mean by that term.